toolkit/tools/pkg/imagecustomizerlib/customizepackages_rpm.go

```diff
@@ -0,0 +1,169 @@
+package imagecustomizerlib
+
+import (
+	"context"
+
+	"github.com/microsoft/azurelinux/toolkit/tools/imagecustomizerapi"
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/safechroot"
+)
+
+// Distribution Configurations
+
+// azureLinuxDistroConfig implements distroHandler for Azure Linux
+type azureLinuxDistroConfig struct {
```

Comment:
Each class (i.e. a struct with methods) should have its own file. Structs that only store data are not required to live in their own file.